src/services/gp2gp/__tests__/send-core.test.js

```diff
@@ -50,7 +48,7 @@ describe('sendCore', () => {
 
   it('should throw error if sending ehr core unsuccessful', async () => {
     // when
-    const mockUrlRequest = nock(REQUEST_BASE_URL, HEADERS)
+    nock(REQUEST_BASE_URL, HEADERS)

```

Comment:
Same as above, just checking we don't need to verify that the URL Scope is done! 😄